app/instance-initializers/bugsnag.js

```diff
@@ -1,7 +1,7 @@
 import Ember  from 'ember';
 import config from '../config/environment';
 import { getContext, generateError } from 'ember-cli-bugsnag/utils/errors';
-import { getMetaData } from '../utils/bugsnag';
+import { getMetaData, getUser } from '../utils/bugsnag';
```

Comment:
Agreed that's definitely a problem. I'm relatively offended by the nonstandard filename anyway.

In an ideal world what do you think the API should be for specifying custom methods like `getMetaData` and `getUser`. It seems like an event listener kind of pattern to me.